docs/references/cmcspec.mdx

```diff
@@ -0,0 +1,552 @@
+        -   Create Canister Refund Fee: `10_000` e8s (`CREATE_CANISTER_REFUND_FEE`)
+        -   Top Up Refund Fee: `10_000` e8s (`TOP_UP_CANISTER_REFUND_FEE`)
+        -   Mint Cycles Refund Fee: `10_000` e8s (`MINT_CYCLES_REFUND_FEE`)
+    -   The `reason` field provides details. The `block_index` indicates the ledger block of the successful refund transfer, if applicable.
+-   **`InvalidTransaction`**: The provided `block_index` might correspond to a transaction that doesn't match the request (e.g., wrong memo for the operation, wrong destination subaccount) or was already processed under a *different* `notify_*` call.
+-   **`Processing`**: Indicates contention. Simply retrying the *exact same call* after a short delay is appropriate.
+-   **`TransactionTooOld`**: The CMC only keeps a history of recent notifications (`MAX_NOTIFY_HISTORY` entries, currently 1,000,000). Older transactions cannot be processed. The returned value is the oldest `block_index` the CMC *might* still know about.
+-   **`Other`**: Catches various issues, such as failure to fetch the block from the ledger (`FailedToFetchBlock` error code `1`), internal CMC errors, or authorization issues (`Unauthorized` error code `3` if `notify_create_canister` caller != creator and is not NNS Dapp), bad subnet selection (`BadSubnetSelection` error code `4`).
+
+## `create_canister`
+
+Creates a new canister using cycles attached directly to the call (not from an ICP ledger payment).
+
+Unlike `notify_create_canister`, this method does not rely on an external ICP transaction. Instead, the calling canister must attach enough cycles to cover the creation cost.
+
+-   If `subnet_selection` and the (deprecated) `subnet_type` are omitted, the CMC will select a suitable subnet at random from the available subnets (either default subnets or those specifically authorized for the caller principal).
+-   If `settings` is provided, it will override the default configuration for the new canister.
+-   If no controllers are given in `settings`, the calling principal becomes the sole controller.
+
+#### Parameters
+
+```candid
+record {
+  settings: opt CanisterSettings;       // Optional settings: controller(s), allocations, limits, etc.
+  subnet_type: opt text;                // (Deprecated) Legacy subnet type selection
+  subnet_selection: opt SubnetSelection; // Preferred way to select the subnet
+}
+```
+
+#### Returns
+
+```candid
+variant { Ok: principal; Err: CreateCanisterError }
+```
+
+-   **Ok**: The principal ID of the newly created canister.
+-   **Err**: A `CreateCanisterError` indicating why creation failed. This is currently always a `Refunded` variant:
+    ```candid
+    type CreateCanisterError = variant {
+      Refunded : record { refund_amount: nat; create_error: text; };
+    };
+    ```
+    -   `create_error`: Text description of the failure (e.g., "Insufficient cycles attached", "No subnets in which to create a canister", or errors from the IC management canister).
```

Comment:
do we still call this "IC management canister" even though we should use "ICP" instead of "IC" in most places?